commons/src/test/java/nl/nn/adapterframework/util/DateFormatUtilsTest.java

```diff
@@ -116,4 +120,13 @@ public void testParseAnyDate3() throws Exception {
 		Date date = DateFormatUtils.parseAnyDate("05/10/98 05:47:13");
 		assertEquals(getCorrectedDate(907566433000L), date.getTime());
 	}
+
+
+	@Test
+	public void testInstantInsteadOfDate() {
+		DateFormat format = new SimpleDateFormat(DateFormatUtils.FORMAT_GENERICDATETIME);
```

Comment:
`java.time.format.DateTimeFormatter.of(DateFormatUtils.FORMAT_GENERICDATETIME)`?